feature-toggle-api/templates/tests/test-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
is it worth testing the import and retrieving of a toggle?

Reply:
it is - i will add more relevant test

Reply:
i will revisit this with another PR